clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -13,6 +13,9 @@ import java.util.concurrent.TimeUnit
 import java.util.concurrent.Callable
 
 private object ApiClient {
+  def translateS3String(uri: String): String =
```

Comment:
Can you please document the reason we are translating this (to use s3aFileSystem)?

Reply:
# 👍🏽 

But actually this function was both broken (it translates `s3://a/b/c` into `s3a/a/b/c`) and never used (`getStorageNamespace` already calls `translateS3`).  So I removed this one, and commented on `translateS3`.